app/models/task.rb

```diff
@@ -264,6 +264,41 @@ def do_hold(last_check)
     end
   end
 
+  # Returns if task is the root task or not
+  def root
```

Comment:
This is already defined and we do not want to override it. See https://github.com/mceachen/closure_tree#accessing-data